pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -19,7 +19,7 @@ func NewInspectCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 	opts := &Options{}
 	inspectCmd := &cobra.Command{
 		Use:    "inspect [<file path> | oci://<OCI image URI>] --bundle <path-to-bundle>",
-		Args:   cobra.ExactArgs(1),
+		Args:   cmdutil.MinimumArgs(1, "must specify file path or container image URI, as well --bundle"),
```

Comment:
**Question**

Under what circumstances would you expect more than 1 arg here? I would expect that this only applies to positional args. Were you looking to provide a useful error message if the user did `gh attestation inspect --bundle bundle.json` with no file or image path?

Reply:
Oh, oops, I didn't realize `cmdutil.ExactArgs()` existed - I think that's more appropriate here.